src/radical/pilot/agent/scheduler/base.py

```diff
@@ -976,8 +976,8 @@ def _set_tuple_size(self, unit):
         use it for fast task-to-placement lookups.
         '''
 
-        d = unit['description']
-        unit['tuple_size'] = tuple([d.get('cpu_processes', 1),
+        d = task['description']
+        task['tuple_size'] = tuple([d.get('cpu_processes', 1),
                                     d.get('cpu_threads',   1),
                                     d.get('gpu_processes', 0),
                                     d.get('cpu_process_type')])
```

Comment:
How about `gpu_process_type` and `gpu_threads`? If I remember correctly, we do not handle them, but wouldn't make sense to include them?

Reply:
Lets not do this in this PR...